rules-tests/Php80/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector/Fixture/different_variable_name_used.php.inc

```diff
@@ -27,9 +27,9 @@ namespace Rector\Tests\Php80\Rector\Class_\ClassPropertyAssignToConstructorPromo
 class DifferentVariableNameUsed
 {
     /**
-     * @param int $ID
+     * @param int $id
      */
-    public function __construct(private $ID)
+    public function __construct(private $id)
```

Comment:
new fixture is needed when `public` and `protected` property should not be changed.